code/modules/jobs/job_types/shaft_miner.dm

```diff
@@ -30,6 +62,16 @@
 	job_flags = JOB_ANNOUNCE_ARRIVAL | JOB_CREW_MANIFEST | JOB_EQUIP_RANK | JOB_CREW_MEMBER | JOB_NEW_PLAYER_JOINABLE | JOB_REOPEN_ON_ROUNDSTART_LOSS | JOB_ASSIGN_QUIRKS | JOB_CAN_BE_INTERN
 
 
+/datum/job/shaft_miner/after_spawn(mob/living/spawned, client/player_client)
+	..()
+	RegisterSignal(spawned, COMSIG_MOB_POINTED, PROC_REF(point_speech))
```

Comment:
we're avoiding innate job features, can this be a skillchip or some equipment item?

Reply:
>skilllchip for a meme feature

can we please not set this standard for skillchips